cdc/model/mounter.go

```diff
@@ -83,6 +86,32 @@ func (e *PolymorphicEvent) IsResolved() bool {
 	return e.RawKV.OpType == OpTypeResolved
 }
 
+// SetUpFinishedCh set up the finished chan, should be called before mount the event.
+func (e *PolymorphicEvent) SetUpFinishedCh() {
+	if e.finished == nil {
+		e.finished = make(chan struct{})
```

Comment:
I wonder whether frequently making and closing channel for each event would cause large overhead.